Description.md

```diff
@@ -1,30 +1,21 @@
+Akka Serverless makes development and operation of high-performing stateful services enjoyable: it provides SDKs for building services, and a managed cloud platform for deploying them. The SDKs expose a simple programming model, available in popular programming languages, that eliminates the need for plumbing code to handle database access or connections. The managed platform relieves you from configuring and maintaining the orchestration platform, and the data stores. Akka Serverless auto-scales services, and handles network partitions and failures. It also gives clear visibility into the running system with unified and scalable logging and monitoring.
 
-Cloudstate is a specification, protocol, and reference implementation for providing distributed state management patterns suitable for **Serverless** computing. 
-The current supported and envisioned patterns include:
+Read more [here](https://developer.lightbend.com/docs/akka-serverless/index.html) and sign-up [here](https://console.akkaserverless.com/p/register#) for a free account. 
 
-* **Event Sourcing**
-* **Conflict-Free Replicated Data Types (CRDTs)**
-* **Key-Value storage**
-* **P2P messaging**
-* **CQRS read side projections**
+The Akka Serverless Python user language support is a library that implements the Akka Serverless protocol and offers an pythonistic API 
```

Comment:
```suggestion
The Akka Serverless Python SDKis a library that implements the Akka Serverless protocol and offers an pythonistic API 
```

User Language Support Library is old Cloudstate speak I think.

Reply:
Yes, Akka Serverless is using `SDK`.